fhirpipe/transform/fhir.py

```diff
@@ -110,28 +102,17 @@ def bind_reference(fhir_object, fhir_spec):
     return:
         a fhir_object where valid references are now FHIR uris
     """
-
+    # TODO think again about how to bind references
+    # Need to load resources in
     # First we check that the reference has been provided
     if fhir_object and fhir_object["identifier"]["value"]:
+        # get id
```

Comment:
fhir_object["identifier"] should be a list of identifies right?

Reply:
Right!
Do you think we can loop on all the identifiers and bind to the first one that can be resolved? Or should we at least check that all the identifiers do identify the same target?

Reply:
Hum, both are interesting but I'll suggest starting with the first method - because it's simpler

Reply:
Or maybe the easier is just to bind for all the elements in fhir_object["identifier"] and keep the same structure (list of identifiers)

Reply:
Forget that, identifier is not an array for a reference ^^